capacities.go

```diff
@@ -0,0 +1,93 @@
+package packngo
+
+const capacityBasePath = "/capacity"
+
+// CapacityService interface defines available capacity methods
+type CapacityService interface {
+	List() (*CapacityReport, *Response, error)
+	Check(*CapacityInput) (*Response, error)
+}
+
+// CapacityInput struct
+type CapacityInput struct {
+	Servers []ServerInfo `json:"servers,omitempty"`
+}
+
+// ServerInfo struct
+type ServerInfo struct {
+	Facility string `json:"facility,omitempty"`
+	Plan     string `json:"plan,omitempty"`
+	Quantity int    `json:"quantity,omitempty"`
+}
+
+type capacityRoot struct {
+	Capacity CapacityReport `json:"capacity,omitempty"`
+}
+
+// CapacityReport struct
+type CapacityReport struct {
```

Comment:
instead of lines 26-69, I would do
```
type CapacityPerBaremetal struct {
	Level string `json:"level"`
}
type CapacityReport map[string]map[string]CapacityPerBaremetal
```
.. it's more simple and more resistant to change.

It's possible to test for key existence in map - that can be used to check for typos.